requirements.txt

```diff
@@ -9,3 +9,4 @@ pip~=22.0,<22.1
 pysmiles~=1.0
 pythreejs~=2.1
 widget-periodictable~=3.0
+numpy<1.24
```

Comment:
No other packages pinning the numpy version will lead to the installation of `numpy==1.24`. It cause the issue describe in https://stackoverflow.com/questions/74844262/how-can-i-solve-error-module-numpy-has-no-attribute-float-in-python. The ase use `numpy.float` which deprecate at `1.20` and removed from `1.24`.